packages/contentstack-clone/package.json

```diff
@@ -6,7 +6,7 @@
   "bugs": "https://github.com/rohitmishra209/cli-cm-clone/issues",
   "dependencies": {
     "@contentstack/cli-cm-export": "~1.10.2",
-    "@contentstack/cli-cm-import": "~1.13.0",
+    "@contentstack/cli-cm-import": "~1.14.0",
```

Comment:
@cs-raj , I think in case of minor version we need to do version bump ?

Reply:
Yes @aman19K I changed it to patch version as changes are minor